src/main/webapp/app/exam/participate/exam-cover/exam-participation-cover.component.html

```diff
@@ -33,13 +33,13 @@ <h2 style="text-align: center; font-weight: normal" *ngIf="startView">
         {{ 'artemisApp.studentExam.submissionNotInTime' | artemisTranslate }}
     </div>
     <ng-container *ngIf="!studentFailedToSubmit">
-        <div class="form-check mt-1">
+        <div class="form-check mt-1 ps-0">
             <input
                 [(ngModel)]="confirmed"
                 type="checkbox"
                 id="confirmBox"
                 (click)="updateConfirmation()"
-                class="form-check-input"
+                class="form-check-input ms-0"
```

Comment:
Removing the margin here doesn't look like a good idea, as it removes the spacing between the Exam start confirmation text and the checkbox:

https://user-images.githubusercontent.com/23171488/167206220-d0f1b517-f3dc-4ff4-bc65-5fe0b6a3b49c.mov

Reply:
Good catch, thank you :D Your reviews are always on point! Should be fixed now :+1:

Reply:
This was also present on the end of exam screen, but should be fixed too :+1: